src/core/pathResolver.ts

```diff
@@ -28,11 +28,14 @@ export class PathResolver {
 	 * The caller must ensure this directory exists before use.
 	 */
 	public getBinaryCachePath(label: string): string {
-		const configPath = vscode.workspace
+		const settingPath = vscode.workspace
 			.getConfiguration()
-			.get<string>("coder.binaryDestination");
-		return configPath && configPath.trim().length > 0
-			? path.normalize(configPath)
+			.get<string>("coder.binaryDestination")
+			?.trim();
```

Comment:
Non-blocking: I think we are not trimming `coder.headerCommand` nor `coder.defaultUrl`.  Do you know if VS Code's configuration already trims values?  If so, we can remove this, if not maybe we should add `trim()` to the others as well (eventually).

Reply:
Awesome, thank you!